src/App.jsx

```diff
@@ -1,13 +1,150 @@
+                    <h2>🥝 Kiwi's </h2>
+                    <Counter
+                        setFruitCount={setKiwis}
+                        fruitCount={kiwis}
+                    />
+                </article>
+                <button className="btn-reset-counters" type="button" onClick={() => resetCounters()}>Reset</button>
+                <form className="order-form" onSubmit={handleSubmit}>
+                    <label htmlFor="firstname">Voornaam :
+                        <input
+                            name="firstname"
+                            type="text"
+                            value={formState.firstname}
+                            onChange={handleChange}
+                        />
+                    </label>
+                    <label htmlFor="lastname">Achternaam :
+                        <input
+                            name="lastname"
+                            type="text"
+                            value={formState.lastname}
+                            onChange={handleChange}
+                        />
+                    </label>
+                    <label htmlFor="age">Leeftijd :
+                        <input
+                            name="age"
+                            type="number"
+                            value={formState.age}
+                            onChange={handleChange}
+                        />
+                    </label>
+                    <label htmlFor="postal-code">Postcode :
+                        <input
+                            name="zipcode"
+                            type="text"
+                            value={formState.zipcode}
+                            onChange={handleChange}
+                        />
+                    </label>
+                    <label htmlFor="delivery-frequency">Bezorgfrequentie :</label>
+                    <select
+                        name="deliveryFrequency"
+                        value={formState.deliveryFrequency}
+                        onChange={handleChange}>
+                        <option value="week">iedere week</option>
+                        <option value="two-week">om de week</option>
+                        <option value="month">iedere maand</option>
+                    </select>
+                    <label htmlFor="comments">Opmerkingen</label>
+                    <textarea
+                        name="comment"
+                        value={formState.comment}
+                        onChange={handleChange}
+                        rows={5}
+                        cols={30}
+                    />
+                    <input
+                        name="agreeTerms"
+                        type="checkbox"
+                        value={formState.agreeTerms}
+                        onChange={handleChange}
+                    />
+                    <label htmlFor="terms">
+                        Ik ga akkoord met de voorwaarden
+                    </label>
```

Comment:
```suggestion
                    <span className="flexrow">
                        <input
                            name="agreeTerms"
                            type="checkbox"
                            value={formState.agreeTerms}
                            onChange={handleChange}
                        />
                        <label htmlFor="terms">
                            Ik ga akkoord met de voorwaarden
                        </label>
                    </span>
```

Samen met die css aanpassing moet dit ervoor zorgen dat het netjes naast elkaar staat 😉